src/parser/core/modules/Defensives.tsx

```diff
@@ -115,17 +131,22 @@ export class Defensives extends Analyser {
 
 	private getUsageRow(entry: CooldownHistoryEntry, defensive: Action): ReactNode {
 		return <>
-			<Table.Row key={entry.start}>
-				<Table.Cell>
-					<Trans id="core.defensives.table.usage-row.text">Used at <Button
-						circular
-						compact
-						size="mini"
-						icon="time"onClick={() => this.timeline.show(entry.start - this.parser.pull.timestamp, entry.end - this.parser.pull.timestamp)}>
-					</Button> {this.parser.formatEpochTimestamp(entry.start)}
-					</Trans>
-				</Table.Cell>
-			</Table.Row>
+			{
+				// Only create the usage rown if this history entry was for this cooldown, not another in the same cooldown group
+				entry.action.id === defensive.id ?
```

Comment:
nit; possibly worth a function for the jsx on this just so it doesn't all end up in a big ol' ternary

Reply:
Cleaner, agreed.